src/engine/objects/SimWall.ts

```diff
@@ -0,0 +1,28 @@
+import * as THREE from "three";
+import { SimObject, ISimObjectSpec } from "./SimObject";
+
+export interface ISimWallSpec extends ISimObjectSpec {
+    wallColor?: number;
+}
+export class SimWall extends SimObject {
+    constructor(spec: ISimWallSpec) {
+        super();
+
+        const meshGeom = new THREE.BoxGeometry(spec.extents.xh, spec.extents.yh, spec.extents.zh);
```

Comment:
I’m not sure this is the right size? I’ll need to check, the extents referenced here are half the width of the object.

Reply:
yeah we'll need to tweak this for sure. I'm actually fairly convinced my GridUtil is drawing a grid that is too big as well